website/src/app/kb/authenticate/entra/readme.mdx

```diff
@@ -0,0 +1,262 @@
+import Alert from "@/components/DocsAlert";
+import SupportOptions from "@/components/SupportOptions";
+import PlanBadge from "@/components/PlanBadge";
+import Image from "next/image";
+
+<div className="flex justify-between items-center">
+
+# SSO with Microsoft Entra
+
+<PlanBadge plans={["enterprise"]} />
+</div>
+
+Firezone integrates with Microsoft Entra using a custom connector that supports
+both authenticating users and syncing users and groups to Firezone. Use this
+guide if you're looking to setup SSO with Microsoft Entra for your Firezone
+Enterprise account and want automatic user / group syncing between Microsoft
+Entra and Firezone.
+
+<Alert
+  color="info"
+  html={`
+    If you're just looking to authenticate users against Microsoft Entra
+    <strong>without</strong> automatic user / group sync, use our
+    <a href="/kb/authenticate/oidc" class="text-accent-500 hover:underline">
+      universal OIDC connector
+    </a> instead, available on all plans.
+  `}
+/>
+
+## Overview
+
+The Firezone Microsoft Entra connector integrates with Microsoft's identity APIs
+to support user authentication and user / group sync.
+
+Users, groups, and organizational units are synced every few minutes to ensure
```

Comment:
Let's not give `few minutes` promises, but rather say that we try to sync them at least once an hour or so. This will be better for the future when we have more issues with scaling workers.

Reply:
K

Reply:
Or maybe we can explain the architecture better in the User-group sync docs